web-client/src/pages/admin/AdminInventoryPage.tsx

```diff
@@ -1,5 +1,100 @@
-import UnderConstruction from "@/components/UnderConstruction";
+import { Button } from "@/components/ui/button";
+import { Search } from "lucide-react";
+import { ChevronDown } from "lucide-react";
+import { DropdownMenu, DropdownMenuItem, DropdownMenuTrigger, DropdownMenuContent } from "@/components/ui/dropdown-menu";
+import InventoryTable from '@/components/InventoryTable';
+import { useNavigate } from 'react-router-dom';
+import { useGetSocks } from '@/api/inventory/queries';
+import { useState } from 'react';
 
 export default function AdminInventoryPage() {
-  return <UnderConstruction pageName="Inventory" />;
+  const navigate = useNavigate();
+
+  const [page, setPage] = useState(1);
+  const limit = 8;
+  const offset = (page - 1) * limit;
+
+  const { data, error, isLoading } = useGetSocks(limit, offset);
+
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+
+  if (error) {
+    return <div>Error fetching socks: {error instanceof Error ? error.message : 'Unknown error'}</div>;
+  }
+
+  if (!data || !data.items || data.items.length === 0) {
+    return <div>No socks available in the inventory.</div>;
+  }
+
+  const handleRowClick = (sockId: string) => {
+    navigate(`/socks/${sockId}`);
+  };
+
+  const totalSocks = data.total;
+  const totalPages = Math.ceil(totalSocks / limit);
+
+  const handleNextPage = () => {
+    if (page < totalPages) setPage((prev) => prev + 1);
+  };
+  const handlePreviousPage = () => {
+    if (page > 1) setPage((prev) => prev - 1);
+  };
+
+  const sortedSocks = data.items.sort((a, b) => a.id - b.id);
```

Comment:
I think you can remove the sorting from the UI/frontend since it could be deceitful to the user.

In the backend, the `GET /socks` endpoint returns the results sorted by `created_at` which should be fine.

Reply:
@sebastian-nunez  I had added the sortedSocks since sometimes when I would change the value of the socks_results_limit the sock with the sockID of 1 would appear at the bottom of the first page.

So for example, if i changed the limit to 6 socks per page the first page would have the socks in order of 2,3,4,5,6,1

Reply:
Yea that's fine actually since all of the socks are pretty much inserted at the exact same moment into the DB they share the same created_at timestamp. But if you were to insert a newer sock, it should appear on top since it's the most recent.

Reply:
Ok i'll remove it then!

Reply:
Updated:
https://github.com/sockify/sockify/pull/120/commits/54bd79515bbe006822b582906e2da97e4c6fbd33